app/components/eventSearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,20 @@
+
+
+export default function SearchBar() {
```

Comment:
@sinhayash9546 This will be a generic search bar that we can use anywhere in the project. Hence I sugegst the following changes:
- the folder name should be `searchBar`
- the file name should be `index.tsx` (to follow uniformity)
- make it a functional and controlled component, it will need the following props:
-- placeholder (default = `Search`)
-- searchInput (this will be used be parent to control the value property of textfield)
-- onChange (this will be used by parent to update the search state)